src/components/Categories.js

```diff
@@ -1,10 +1,12 @@
 import '../styles/categories.css';
+import { useState } from 'react';
 
 const Categories = () => {
-  const notReady = 'Under Construction';
+  const [status, setStatus] = useState('Page Under Construction');
+
   return (
     <div className="CategoriesDiv">
-      <p>{notReady}</p>
+      <div>{status}</div>
     </div>
```

Comment:
- [ ] Great job, here we can use our state to add a button inside our component and when clicked we dispatch an action that checks the status and display the status on the page.